support/Pipelines/Bbh/InitialData.py

```diff
@@ -369,5 +447,52 @@ def generate_id_command(
     )
 
 
+@click.command(name="postprocess-id", help=postprocess_id.__doc__)
+@click.argument(
+    "id_input_file_path",
+    type=click.Path(
```

Comment:
Help text

Reply:
Positional arguments don't have a `help`, it's part of the main help text